tests/hack/ci/Makefile

```diff
@@ -52,12 +53,14 @@ create-kyma-system-ns: ## Create kyma-system namespace.
 create-k3d: ## Create k3d with kyma CRDs.
 	@echo "::group::create-k3d"
 	k3d cluster create ${CLUSTER_NAME} \
+		--image ${K3S} \
 		--api-port 6550 \
 		-p 8080:80@loadbalancer \
 		-p 8443:443@loadbalancer \
 		-p 9090-9099:9090-9099@loadbalancer \
 		--agents 2 \
 		--registry-create k3d-${REGISTRY_NAME}:${REGISTRY_PORT}
+	kubectl version
```

Comment:
was your intention to leave this version print?

Reply:
I thought it doesn't hurt and lets us inspect versions nicely. But the k3s version is also visible in the image param, and client isn't _that_ important, so imo either way is good, I can remove it if you feel it adds nothing.

Reply:
thanks. let's leave it as it is